presto-hive/src/test/java/io/prestosql/plugin/hive/s3/TestHiveS3Config.java

```diff
@@ -109,6 +111,7 @@ public void testExplicitPropertyMappings()
                 .setS3PathStyleAccess(true)
                 .setS3UseInstanceCredentials(false)
                 .setS3IamRole("roleArn")
+                .setS3StorageClass(PrestoS3StorageClass.INTELLIGENT_TIERING)
```

Comment:
I guess we can import the enum directly

Reply:
@Praveen2112 Importing Enum directly might look off since `PrestoS3SignerType` and `PrestoS3AclType` are used without importing Enum directly.